server/src/services/mail/index.js

```diff
@@ -0,0 +1,32 @@
+import nodemailer from "nodemailer";
+import { config } from "dotenv";
+config();
+
+const sendEmail = () =>{
+
+    let transporter = nodemailer.createTransport({
+        host: "smtp.mailtrap.io",
+        port: 2525,
+        auth: {
+          user: process.env.MAIL_USER,
+          pass: process.env.MAIL_USER_PASSWORD
+        }
+    });
+  
+    let mailOptions = {
```

Comment:
```suggestion
    const mailOptions = {
```

Reply:
It is better to create const instead of lets, beacuse better avoid mutable variables when this is not needed